cpp_testing/modules/CMakeLists.txt

```diff
@@ -1,19 +1,19 @@
 #****************************************************************************
 #
 #                           Klepsydra Core Modules
-#              Copyright (C) 2019-2020  Klepsydra Technologies GmbH
+#              Copyright (C) 2019-2031  Klepsydra Technologies AG
```

Comment:
move this repo to PE?

Reply:
No, it's required for parts of kpsr-core or the kpsr-tutorial.